Chapters/3-3-Chapter17.md

```diff
@@ -0,0 +1,271 @@
+
+"Well look who it is," she scoffed as the group approached her, "The traitor and her band of thieves."
+
+Pomri responded with a fake smile, "Such a warm welcome, Bavle."
+
+"Oh, pardon my manners," she said facetiously. She pointed to the wetlands below them. "Perhaps you'd like a drop of water first."
+
+Pomri banged her spear on the bridge. "Perhaps you'd like this spear down your throat, ya cheeky cunt."
+
+"Pomri, what's this about?" Melna whispered.
+
+But Bavle cut her off, "So tell me why I should let you bastards through."
+
+A lull in the conversation formed while Melna watched their eyes turn toward her. It took a moment for her to realize, then she took a quick breath to begin, "Oh. We are on our way to a senate session. We would like to pass."
+
+"No," said the guard.
+
+"What? Why?"
+
+"Well, you haven't told me the password."
+
+"A password?"
+
+"Oh yes. Every city has them ever since The Band of Thieves infiltrated the gates of Yansha."
+
+Melna paused, trying to understand such a ridiculous requirement. _Since when was there a password to enter Rudjan? And who is The Band of Thieves? Why did nobody brief me on them?!_. She thought deeply for a long and awkward moment, pondering what the password could be, when all of a sudden she heard a chuckle coming from Pomri, and then Bavle, and then boisterous laughing from the two.
+
+Pomri exclaimed, "Oi! A password? The Band of Thieves? Really, Melna?"
+
+Bavle joined her, "Are all your fidelants so dull?"
+
+Melna was unamused at being the brunt of a joke, not to mention the waste of precious time, but she took whatever humor they could afford.
+
+"Very funny," she said. A glare shot from Melna's eyes toward the still-laughing Pomri, and she quieted down and helped Linyu off of Laen-Laen.
+
+"We have a train to catch," said Melna, "so please let us be on our way."
+
+Bavle approached the saffa while she continued with her friend, "Oh I've missed your ugly face, Pomri. Come back when you're done running around with this dullard."
+
+Melna cleared her throat, uneasy with how brazenly they insulted her.
+
+"Count on it," said Pomri, "Just gotta save the world first."
+
+"Just another day for you," said Bavle with a voice almost motherly. "Well, the dockways aren't big enough for saffas so you'll have to leave them here. They'll be safe on these wetlands as long as they're well trained. And I assume since you made it this far, that won't be an issue?"
+
+"Yes, of course," said Melna, impatient to end the conversation.
+
+"Laen," Linyu whispered to him, keeping a tight hold on his fur.
+
+Melna held her impatience for a moment, for Linyu's sake.
+
+"He'll be fine here," she said. "This wetland is full of saffas and tasty things. This is where saffas come from, after all."
+
+Linyu moved hae hands to Laen-Laen's neck, and scratched him gently, to Laen's chagrin, while Melna faced him and said her own kind of goodbye.
```

Comment:
Once upon a time, a very bad lady (Stephanie Meyer) misused the word "chagrin". And everyone talked about the misuse of that word as an example how bad the prose was. Since then it entered into public consciousness; "chagrin" is a word that makes people cringe and think of Twilight. It may be best to choose another word because sparkling vampires is something we'll like to collectively forget.